### DISCUSSION TITLE
功能探讨 关于使用蓝牙同步

### DISCUSSION THREAD

**Discussion by LunacyZeus** [Jeric-X/SyncClipboard#92] [Ideas]
我在想有没有这个功能 把同步服务跑在esp32上 通过蓝牙/wifi模拟设备的形式 完成多设备数据同步
当前设备之间要在一个局域网（wifi）
离得很近（蓝牙）

**Comment by LunacyZeus** [comment]
只是无聊的想法 没其他的意思 我先用手上的硬件测测看

**Comment by Jeric-X** [comment]
这种想法很好啊，特别是手边有硬件的小伙伴都可以尝试/练手，不过不管是单片机还是蓝牙网络我都不是很懂，如果有任何我可以讨论的问题都可以继续探讨
